occurrences/schema.py

```diff
@@ -98,9 +98,50 @@ class PalvelutarjotinEventInput(InputObjectType):
     auto_acceptance = graphene.Boolean()
 
 
-class StudyGroupNode(DjangoObjectType):
-    study_level = StudyLevelEnum()
+class StudyLevelTranslationType(DjangoObjectType):
+    language_code = LanguageEnum(required=True)
 
+    class Meta:
+        model = StudyLevelTranslation
+        exclude = ("id", "master")
+
+
+class StudyLevelTranslationsInput(InputObjectType):
+    language_code = LanguageEnum(required=True)
+
+
+class StudyLevelNode(DjangoObjectType):
+    # NOTE: Should the originalId have a better name?
+    originalId = graphene.String(source="pk")
+    label = graphene.String(
+        description="Translated field in the language defined in request "
+        "ACCEPT-LANGUAGE header "
+    )
+
+    class Meta:
+        model = StudyLevel
+        interfaces = (relay.Node,)
+        exclude = ("study_groups",)
+
+    @classmethod
+    def get_queryset(cls, queryset, info):
+        lang = get_language()
+        return queryset.language(lang)
+
+    @classmethod
+    def get_node(cls, info, id):
+        return super().get_node(info, id)
```

Comment:
Codecov says it was not covered by tests, but is this method even needed?

Reply:
True, I think you can just remove this. I think this also presents somewhere in my old code :P